test/system/260-sdnotify.bats

```diff
@@ -147,33 +156,43 @@ READY=1" "sdnotify sent MAINPID and READY"
     _start_socat
 
     run_podman run -d --sdnotify=container $_FEDORA \
-               sh -c 'printenv NOTIFY_SOCKET;echo READY;systemd-notify --ready;while ! test -f /stop;do sleep 0.1;done'
+               sh -c 'printenv NOTIFY_SOCKET; echo READY; while ! test -f /stop;do sleep 0.1;done;systemd-notify --ready'
```

Comment:
This terrifies me. As I see it, the purpose of `systemd-notify` is to tell a parent "hey, I'm done loading the container and am ready to do some work. Feel free to interact with me." By doing `systemd-notify` then immediately exiting, this is the complete opposite.

Reply:
I agree that exiting right after sending READY isn't representative. But I think the test is now more accurate than before.  When using the "container" policy, the MAINPID and READY message are not sent simultaneously; the test did not allow for testing that before.

@edsantiago, I could add a second loop waiting for another file if you want to.

Reply:
(Still thinking about it. Will write back after scrum. Thanks for your feedback.)

Reply:
Thanks again. I'll step back and just trust that this all works.